dotnet/samples/Concepts/Agents/ChatCompletion_FunctionTermination.cs

```diff
@@ -23,7 +23,7 @@ public async Task UseAutoFunctionInvocationFilterWithAgentInvocationAsync()
             {
                 Instructions = "Answer questions about the menu.",
                 Kernel = CreateKernelWithFilter(),
-                Arguments = new KernelArguments(new OpenAIPromptExecutionSettings() { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions }),
+                Arguments = new KernelArguments(new OpenAIPromptExecutionSettings() { FunctionChoiceBehavior = FunctionChoiceBehavior.Auto() }),
```

Comment:
How do we avoid confusion for developers?

The old `ToolCallBehavior` still works, so somewhere (Microsoft Learn) we need to add a statement explain that there is a new recommended why to configure function calling. We need to clearly explain the advantages of the new approach and the timeline to obsolete the old approach.